tests/nonlinear_channel_flow_tractions_dc_picard_Stokes_GMG.prm

```diff
@@ -0,0 +1,19 @@
+#Test GMG  iterated defect correction Stokes solver based on
+# benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm
+
+include $ASPECT_SOURCE_DIR/benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/input_t.prm
+set Additional shared libraries = $ASPECT_SOURCE_DIR/benchmarks/newton_solver_benchmark_set/nonlinear_channel_flow/libsimple_nonlinear.so
```

Comment:
Remove this line. Instead, you need to add a .cc with the same name as the prm that contains an #include of the cc file. Take a look at similar tests.